### PR TITLE
Calculate TVOD data

### PR DESCRIPTION
**Pull request by zaher-aa**
- Calculate TVOD cards count
- Calculate TVOD cards watched seconds
- Store data to the DB

### CODE REVIEW

**Review by Ibrahim-Jarada — APPROVED** [review]

File: src/utilities/viewlift/gqlQueries/index.ts [hunk 1]
```diff
@@ -1,2 +1,4 @@
 export { default as listUserWatchedQuery } from './listUserWatchedQuery';
 export { default as loginMutation } from './loginMutation';
+export { default as listSubscriptionsQuery } from './listPaymentsQuery';
```
Comment (Ibrahim-Jarada) [inline src/utilities/viewlift/gqlQueries/index.ts:3]:
```suggestion
export { default as listPaymentsQuery} from './listPaymentsQuery';
```

File: src/utilities/viewlift/requests/listPayments.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    Authorization,
+    viewliftEndpoint,
+    offset = 0,
+    limit,
+    after,
+  }: any): Promise<any> => {
+    try {
+      const { data: { data: { listPayments: { billingHistory } } } } = await axios
+        .post(viewliftEndpoint, {
+          query: listSubscriptionsQuery,
```
Comment (Ibrahim-Jarada) [inline src/utilities/viewlift/requests/listPayments.ts:26]:
```suggestion
          query: listPaymentsQuery,
```

File: src/utilities/watchesPerContent.ts [hunk 3]
```diff
@@ -1,17 +1,37 @@
...
+
+  paymentsList.forEach(({ contentId, monetizationModel }) => {
+    if (monetizationModel === 'tvod') {
+      if (tvodCount[contentId]) {
+        tvodCount[contentId] += 1;
+      } else {
+        tvodCount[contentId] = 1;
+      }
+    }
+  });
```
Comment (Ibrahim-Jarada) [inline src/utilities/watchesPerContent.ts:25]: Check the payment type giving +1 for "PAYMENT" and -1 for "REFUND"

**Review by Jood80 — APPROVED** [review]
LGTM

File: src/utilities/viewlift/requests/listPayments.ts [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+const getAllPayments = async (
+  listPaymentsData: {
+    Authorization: string,
+    viewliftEndpoint: string,
+    offset: number,
+    limit: number,
+    after: string,
+  },
+): Promise<{ paymentsList:any[] }> => {
+  let paymentsList: any[] = [];
```
Comment (Jood80) [inline src/utilities/viewlift/requests/listPayments.ts:14]: Please make sure to create a proper interface for `paymentsList`.

**Pull request merged**